diff --git a/docroot/.htaccess b/docroot/.htaccess
index efce1877..bf091a87 100755
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -150,6 +150,17 @@ AddEncoding gzip svgz
   # Copy and adapt this rule to directly execute PHP files in contributed or
   # custom modules or to run another PHP application in the same directory.
   RewriteCond %{REQUEST_URI} !/core/modules/statistics/statistics.php$
+
+  #
+  #
+  #
+  # UWM Image caching:
+  # Allow access to simplesaml paths
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+  #
+  #
+  #
+  
   # Deny access to any other PHP files that do not match the rules above.
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
@@ -177,6 +188,19 @@ AddEncoding gzip svgz
       # Force proxies to cache gzipped & non-gzipped css/js files separately.
       Header append Vary Accept-Encoding
     </FilesMatch>
+    
+    #
+    #
+    #
+    # UWM Image caching:
+    <FilesMatch "\.(jpg|jpeg|png|gif|swf)$">
+        # Set images to expire in one year
+        Header set Cache-Control "max-age=31536000, public"
+    </FilesMatch>
+    #
+    #
+    #
+    
   </IfModule>
 </IfModule>
 
