diff --git a/docroot/.htaccess b/docroot/.htaccess
index efce1877..3b451609 100755
--- a/docroot/.htaccess
+++ b/docroot/.htaccess
@@ -1,3 +1,640 @@
+
+
+
+####
+## UWM: Redirect these pages
+## to their canonical sites:
+##
+####
+<IfModule mod_rewrite.c>
+
+    RewriteEngine on
+
+    # Redirect Right as Rain to Live URL
+    RewriteCond %{HTTP_HOST} chew.cms.uwmedicine.org [NC,OR]
+    RewriteCond %{HTTP_HOST} rightasrain.uw.edu [NC]
+    RewriteRule ^(.*)$ https://rightasrain.uwmedicine.org%{REQUEST_URI}  [L,R=301]
+
+    # Redirect Huddle to Live URL
+    RewriteCond %{HTTP_HOST} huddle.cms.uwmedicine.org [NC]
+    RewriteRule ^(.*)$ https://huddle.uwmedicine.org%{REQUEST_URI}  [L,R=301]
+
+    # Redirect naked airliftnw.org to www version
+    RewriteCond %{HTTP_HOST} ^airliftnw.org [NC]
+    RewriteRule ^(.*)$ https://www.%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+    # Redirect airlift-nw.org/anything to uwmedicine.org/airlift-nw/anything
+    RewriteCond %{HTTP_HOST} www.airliftnw.org [NC]
+    RewriteRule ^(.*)$ https://www.uwmedicine.org/airliftnw%{REQUEST_URI} [L,R=301]
+
+</IfModule>
+
+
+
+####
+## UWM: Redirect most domains
+## to include www, https, and no trailing slash,
+## trying to avoid multiple redirects:
+##
+####
+<IfModule mod_rewrite.c>
+
+    RewriteEngine on
+
+    ## Skip this IfModule block
+    ## if using a .local host
+    #RewriteCond %{HTTP_HOST} !^(stevie\.)?uwmed\.local$
+
+    ## For uwmedicine.org, if a trailing slash, remove it and
+    ## go to https://www
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond %{HTTP_HOST} ^uwmedicine\.org [NC,OR]
+    RewriteCond %{HTTP_HOST} ^www\.uwmedicine\.org [NC]
+    RewriteCond %{REQUEST_URI} ^(.+)/$
+    RewriteRule ^(.*)\/{1}(\?.*)?$ https://www.uwmedicine.org/$1$2 [L,R=301]
+
+    ##
+    ## Test version:
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond %{HTTP_HOST} ^uwmedicinetest\.org [NC,OR]
+    RewriteCond %{HTTP_HOST} ^www\.uwmedicinetest\.org
+    RewriteCond %{REQUEST_URI} ^(.+)/$
+    RewriteRule ^(.*)\/{1}(\?.*)?$ https://www.uwmedicinetest.org/$1$2 [L,R=301]
+
+    ## For uwmedicine.org, if no www,
+    ## go to https://www
+    RewriteCond %{HTTP_HOST} ^uwmedicine\.org [NC]
+    RewriteRule ^(.*)$ https://www.uwmedicine.org/$1 [L,R=301]
+    ##
+    ## Test version:
+    RewriteCond %{HTTP_HOST} ^uwmedicinetest\.org [NC]
+    RewriteRule ^(.*)$ https://www.uwmedicinetest.org/$1 [L,R=301]
+
+    ## For all hosts, if a trailing slash, remove it and
+    ## go to https://
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteCond %{REQUEST_URI} (.+)/$
+    RewriteRule ^(.*)/$ https://%{HTTP_HOST}/$1[L,R=301]
+
+    ## For all hosts, if not on https,
+    ## go to https://
+    RewriteCond %{HTTPS} off
+    RewriteCond %{HTTP:X-Forwarded-Proto} !https
+    RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301,NE]
+
+</IfModule>
+
+
+
+####
+## UWM: Provide friendly redirects
+## for certain pages and URLs
+##
+####
+<IfModule mod_rewrite.c>
+
+    RewriteEngine on
+
+    # Route non-production requests to the blocking robots.txt:
+    RewriteCond %{ENV:AH_SITE_ENVIRONMENT} !prod
+    RewriteRule ^robots.txt robots_block.txt [L]
+
+    # For www.uwmedicine,org, redirect old marketing urls:
+    RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+
+    # Redirect education pages to their new site
+    # RewriteRule ^education/?$ https://education.uwmedicine.org/ [L,R=301]
+    # RewriteRule ^education/(.*)$ https://education.uwmedicine.org/$1 [L,R=301]
+
+    # Redirect these urls to their new, page tab urls:
+    RewriteRule ^locations/(.*)/directions(.*) locations/$1#directions-tab [NE,L,R=301]
+    RewriteRule ^locations/(.*)/care-providers(.*) locations/$1#providers-tab [NE,L,R=301]
+    RewriteRule ^services/urgent-care/(providers|locations)$ services/urgent-care [L,R=301]
+    RewriteRule ^services/urgent-care/(.+)#911-calls$ services/urgent-care#block-emergencymedicine [NE,L,R=301]
+
+    # RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1 [L,R=301]
+    # RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+    # RewriteRule ^locations/urgent-care-(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+    # RewriteRule "^/search#k=(.+)$" "https://www.uwmedicine.org/search?s=$1" [NE,L,R=301]
+    # RewriteRule "^/search/providers#k=(.+)$" "https://www.uwmedicine.org/search/providers?s=$1" [NE,L,R=301]
+    # RewriteRule "^/search/locations#k=(.+)$" "https://www.uwmedicine.org/search/locations?s=$1" [NE,L,R=301]
+    # RewriteRule ^/node/255156$ https://www.uwmedicine.org/search/providers?s=&fs_p%5B0%5D=medical-service%3A10566 [NE,L,R=301]
+
+</IfModule>
+
+
+####
+## UWM: Provide an alias
+## for special microsites in /lp/ directory
+## so /lp/ is not required in the URL
+##
+####
+<IfModule mod_rewrite.c>
+
+    RewriteEngine on
+
+    RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+    RewriteCond %{HTTP_HOST} ^www.uwmedicinetest.org
+
+    RewriteRule ^/birth-facility(.*)?$ /lp/birth-facility$1 [NC,L]
+    RewriteRule ^/breakthroughs(.*)?$ /lp/breakthroughs$1 [NC,L]
+    RewriteRule ^/lung-cancer(.*)?$ /lp/lung-cancer$1 [NC,L]
+    RewriteRule ^/uw-baby-pregnancy-app(.*)?$ /lp/uw-baby-pregnancy-app$1 [NC,L]
+
+</IfModule>
+
+
+
+####
+## UWM: Provide a proxy to
+## serve all pages using the Stevie site
+##
+####
+<IfModule mod_rewrite.c>
+
+    RewriteEngine on
+
+    RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+    RewriteCond %{HTTP_HOST} ^www.uwmedicinetest.org
+    RewriteRule ^(.*)$ https://stevie.cms.uwmedicine.org%{REQUEST_URI} [L,P]
+
+</IfModule>
+
+
+
+##
+#
+# Begin old UW Medicine special .htaccess rules
+#
+#
+##
+<IfModule mod_rewrite.c>
+  RewriteEngine on
+
+  ### Redirect ALL education pages to //education.uwmedicine.org ###
+#  RewriteRule ^education/?$         https://education.uwmedicine.org/ [L,R=301]
+#  RewriteRule ^education/(.*)$         https://education.uwmedicine.org/$1 [L,R=301]
+
+#
+#  ## SSL RULES ##
+#  # Redirect HTTP to HTTPS in Acquia Cloud
+#  RewriteCond %{ENV:AH_SITE_ENVIRONMENT} (ra|dev|test|prod)
+#  RewriteCond %{HTTPS} off
+#  RewriteCond %{HTTP:X-Forwarded-Proto} !https
+#  RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+#
+#  # Set "protossl" to "s" if we were accessed via https://.  This is used later
+#  # if you enable "www." stripping or enforcement, in order to ensure that
+#  # you don't bounce between http and https.
+#  RewriteRule ^ - [E=protossl]
+#  RewriteCond %{HTTPS} on
+#  RewriteRule ^ - [E=protossl:s]
+#
+#  # Make sure Authorization HTTP header is available to PHP
+#  # even when running as CGI or FastCGI.
+#  RewriteRule ^ - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+#
+#  ## UWMED CUSTOM REWRITES ##
+#
+#  # Allow simplesaml and UW NetId sign-ons to pass through
+#  RewriteCond %{REQUEST_URI} !^/simplesaml
+#
+#  # Redirect CHEW Dev URL and CHEW UW URL to Live URL
+#  RewriteCond %{HTTP_HOST} chew.cms.uwmedicine.org [OR]
+#  RewriteCond %{HTTP_HOST} rightasrain.uw.edu
+#  RewriteRule ^(.*)$ https://rightasrain.uwmedicine.org%{REQUEST_URI} [L,R=301]
+#
+#  # Redirect Huddle Dev URL to Live URL
+#  RewriteCond %{HTTP_HOST} huddle.cms.uwmedicine.org
+#  RewriteRule ^(.*)$ https://huddle.uwmedicine.org%{REQUEST_URI} [L,R=301]
+#
+#  # Redirect naked airliftnw.org to www version
+#  RewriteCond %{HTTP_HOST} ^airliftnw.org
+#  RewriteCond %{HTTP_HOST} !^www\. [NC]
+#  RewriteRule ^ http%{ENV:protossl}://www.%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+#
+#  # Redirect airlift-nw.org/anything to uwmedicine.org/airlift-nw/anything
+#  RewriteCond %{HTTP_HOST} www.airliftnw.org
+#  RewriteRule ^(.*)$ https://www.uwmedicine.org/airliftnw%{REQUEST_URI} [L,R=301]
+#
+#  # Redirect naked uwmedicine.org and uwmedicinetest.org to www version
+#  RewriteCond %{HTTP_HOST} ^uwmedicine(test)?.org
+#  RewriteCond %{HTTP_HOST} !^www\. [NC]
+#  RewriteRule ^ http%{ENV:protossl}://www.%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+
+
+
+  #########################
+  ### UWMEDICINE LAUNCH ###
+  #########################
+
+  ### Before we do any proxying, handle these redirects ###
+  # Pattern-based redirects for launched pages. These pages are served from Drupal.
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+
+#  ## Provide these folders from the /lp/ directory without lp in the URL: ##
+#  RewriteRule ^/birth-facility(.*)?$ /lp/birth-facility$1 [NC,L]
+#  RewriteRule ^/breakthroughs(.*)?$ /lp/breakthroughs$1 [NC,L]
+#  RewriteRule ^/lung-cancer(.*)?$ /lp/lung-cancer$1 [NC,L]
+#  RewriteRule ^/uw-baby-pregnancy-app(.*)?$ /lp/uw-baby-pregnancy-app$1 [NC,L]
+#
+#  ## Other rewrites: ##
+#  RewriteRule ^locations/(.*)/directions(/)?(?(2).*|$) "https://www.uwmedicine.org/locations/$1#directions-tab" [NE,L,R=301]
+#  RewriteRule ^locations/(.*)/care-providers(/)?(?(2).*|$) "https://www.uwmedicine.org/locations/$1#providers-tab" [NE,L,R=301]
+#  RewriteRule ^services/urgent-care/(providers|locations)$ "https://www.uwmedicine.org/services/urgent-care" [L,R=301]
+#  RewriteRule "^services/urgent-care/(.+)#911-calls$" "https://www.uwmedicine.org/services/urgent-care#block-emergencymedicine" [NE,L,R=301]
+#  RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+#  RewriteRule ^services/urgent-care/(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+#  RewriteRule ^locations/urgent-care-(.+)$ "https://www.uwmedicine.org/locations/primary-care-$1" [L,R=301]
+#  RewriteRule "^/search#k=(.+)$" "https://www.uwmedicine.org/search?s=$1" [NE,L,R=301]
+#  RewriteRule "^/search/providers#k=(.+)$" "https://www.uwmedicine.org/search/providers?s=$1" [NE,L,R=301]
+#  RewriteRule "^/search/locations#k=(.+)$" "https://www.uwmedicine.org/search/locations?s=$1" [NE,L,R=301]
+#  RewriteRule ^/node/255156$ https://www.uwmedicine.org/search/providers?s=&fs_p%5B0%5D=medical-service%3A10566 [NE,L,R=301]
+
+### OLD EDUCATION REDIRECTS SERVED FROM ww2 ###
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+  RewriteCond %{HTTP_HOST} ^www.uwmedicinetest.org
+
+  RewriteCond %{REQUEST_URI} ^/education(/)?(?(1).*|$)$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/default.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/diversity$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/diversity$
+  RewriteCond %{REQUEST_URI} !^/education/about/mission$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/curriculum-preview$
+  RewriteCond %{REQUEST_URI} !^/education/pages/md-program/clinical-curriculum.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/the-college$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/md-program/service-learning.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/wwami/wwami-community$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/wwami$
+  RewriteCond %{REQUEST_URI} !^/education/wwami$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/current-students/curriculum/programs/pathways$
+  RewriteCond %{REQUEST_URI} !^/education/pages/md-program/financial-aid.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/current-students/student-affairs/financial-aid$
+  RewriteCond %{REQUEST_URI} !^/financialaid$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/applications$
+  RewriteCond %{REQUEST_URI} !^/admissions$
+  RewriteCond %{REQUEST_URI} !^/education/md-phd-program$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/expectations$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/deadlines$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/course-requirements$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/applications$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/interview$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/admissions-resources$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/stats$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/decision$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/faqs$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/advisors$
+  RewriteCond %{REQUEST_URI} !^/education/about$
+  RewriteCond %{REQUEST_URI} !^/education/about/academic-departments$
+  RewriteCond %{REQUEST_URI} !^/education/about/administration$
+  RewriteCond %{REQUEST_URI} !^/education/about/department-chairs$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/why-uwsom$
+  RewriteCond %{REQUEST_URI} !^/education/about/willed-body-program$
+  RewriteCond %{REQUEST_URI} !^/schoolofmedicine$
+  RewriteCond %{REQUEST_URI} !^/willedbody$
+  RewriteCond %{REQUEST_URI} !^/education/about/awards$
+  RewriteCond %{REQUEST_URI} !^/education/about/diversity-commitment$
+  RewriteCond %{REQUEST_URI} !^/education/about/mission$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/contact-us$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/md-program/financial-aid-contacts.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/wwami/wwami-contact-us$
+  RewriteCond %{REQUEST_URI} !^/education/gme/contact$
+  RewriteCond %{REQUEST_URI} !^/education/about/willed-body-program/contact-information$
+  RewriteCond %{REQUEST_URI} !^/gme-survey$
+  RewriteCond %{REQUEST_URI} !^/education/phd-program$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/advice$
+  RewriteCond %{REQUEST_URI} !^/education/wwami/wwami-conferences/gme-summit$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/wwami/wwami-conferences/gme-summit$
+  RewriteCond %{REQUEST_URI} !^/GMESummit$
+  RewriteCond %{REQUEST_URI} !^/education/other-programs$
+  RewriteCond %{REQUEST_URI} !^/education/undergraduate-program$
+  RewriteCond %{REQUEST_URI} !^/education/medex$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/physicians$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/advisors$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/applicants/shadowing$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/admissions/directions-lodging$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/us-canadian-students/eligibility$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/us-canadian-students/adding-dropping$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/us-canadian-students/clinical-schedule$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/us-canadian-students/application-deadlines$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/pain-medicine$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/emergency-medicine$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/family-medicine$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/internal-medicine$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/lab-medicine$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/neurological-surgery$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/neurology$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/ob-gyn$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/opthalmology$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/orthopaedics$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/otolaryngology$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/pathology$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/pediatrics$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/psychiatry$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/departmental-courses/radiation-oncology$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/international-students$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/visiting-students/international-students/application$
+  RewriteCond %{REQUEST_URI} !^/LABMED$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/Resources-for-Visiting-Students.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/gme$
+  RewriteCond %{REQUEST_URI} !^/graduate-medical-education$
+  RewriteCond %{REQUEST_URI} !^/education/gme/programs$
+  RewriteCond %{REQUEST_URI} !^/education/gme/residents-fellows$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/gme/uw-nurf.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/gme/prospective-residents$
+  RewriteCond %{REQUEST_URI} !^/education/gme/incoming-residents-and-fellows$
+  RewriteCond %{REQUEST_URI} !^/education/gme/policies-and-procedures$
+  RewriteCond %{REQUEST_URI} !^/education/gme/policies-and-procedures/supervision$
+  RewriteCond %{REQUEST_URI} !^/education/gme/npi-enrollment$
+  RewriteCond %{REQUEST_URI} !^/harborview/Pages/observation-request.aspx$
+  RewriteCond %{REQUEST_URI} !^/uw-medical-center/observership$
+  RewriteCond %{REQUEST_URI} !^/education/gme/residents-fellows/gme-wellness-service$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/Compliance-Vodcasts.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/mistreatment-policy$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/How-to-Access-your-Compliance-Documents.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/current-students/student-affairs/usmle-board-exams$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/usmle-applications-registration$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/usmle-changing-scheduled-test-date$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/implications-if-examination-is-failed$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/current-students/student-affairs/importance-board-exam$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/background-check$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/cbc-incoming-students$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/cbc-continuing-students$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/cpr-for-providers$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/Drug-Testing-for-Select-Clerkship-Rotations.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/Drug-Testing-Procedure.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/hipaa-training$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/data-stewardship$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/universal-precautions-training$
+  RewriteCond %{REQUEST_URI} !^/education/Pages/body-fluid-exposure.aspx$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/e13-compliance-checklist$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/trust-compliance$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/compliance/immunizations$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/academic-progress$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/academic-policies$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/academic-progress/committee$
+  RewriteCond %{REQUEST_URI} !^/education/md-program/academic-policies-and-procedures/academic-progress/guidelines-for-academic-probation$
+
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+
+
+  ### SERVE THESE PAGES FROM www2.uwmedicine.org ###
+  ### These can be removed when SharePoint is shut down ###
+#  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+#
+#  RewriteCond %{REQUEST_URI} ^/dawgscare(/)?(?(1).*|$)$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/we_care(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/obstetrics/uw-baby-pregnancy-app$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/obstetrics/birth-facility$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/obstetrics/PublishingImages(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/flu(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/breakthroughs(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/aco(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/heart-health(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/public_kiosk_links(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/boeing-enrollment(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/lung-cancer(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/Pages/lobby-screen.aspx$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/inolympia(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/insmokeypoint(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/inshoreline(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/inballard(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/som(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/referrals/refer-a-patient(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/PublishingImages(/)?(?(1).*|$) [OR]
+#  RewriteCond %{REQUEST_URI} ^/Style%20Library(/)?(?(1).*|$) [NC,OR]
+#  RewriteCond %{REQUEST_URI} ^/style\ library(/)?(?(1).*|$) [NC,OR]
+#  RewriteCond %{REQUEST_URI} ^/_Layouts(/)?(?(1).*|$) [NC,OR]
+#  RewriteCond %{REQUEST_URI} ^/ScriptResource.axd.* [OR]
+#  RewriteCond %{REQUEST_URI} ^/WebResource.axd.* [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/.*/PublishingImages/.* [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/primary-care-belltown/internal-medicine-residency-program/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/pediatric/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/primary-care-kent-des-moines/residencia-pediatrica/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/primary-care-northgate/family-medicine-residency-program/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/faqs/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/orcas-island-clinic/welcome/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/welcome/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/lopez-island-clinic/faqs/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/urgent-care/tips/? [OR]
+#  RewriteCond %{REQUEST_URI} ^/patient-resources/email-patient(/)?(?(1).*|$)
+#
+#  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+#  RewriteRule ^(.*)$ https://www2.uwmedicine.org%{REQUEST_URI} [L,P]
+
+
+#
+#  ### SERVE THESE PAGES FROM stevie.cms.uwmedicine.org ###
+#  ### These can be removed when cms.uwmedicine.org is decomissioned ###
+#  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+#
+#  RewriteCond %{REQUEST_URI} ^/?$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/primary-care/flu$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/center-for-women-and-children/maternity-care/labor-delivery-tours$ [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/center-for-women-and-children/maternity-care/labor-delivery-tours/obstetrics-tour-registration [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/center-for-women-and-children/preparing-for-you [OR]
+#  RewriteCond %{REQUEST_URI} ^/services/primary-care/behavioral-health-integration-program [OR]
+#  RewriteCond %{REQUEST_URI} ^/locations/harborview-medical-center/centers-of-emphasis$
+#
+#  RewriteRule ^(.*)$ https://stevie.cms.uwmedicine.org%{REQUEST_URI} [L,P]
+
+
+
+  ### SERVE THESE PAGES FROM cms.uwmedicine.org ###
+  ### These can be removed when medical services and make an appointment are served from stevie ###
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org [OR]
+
+  #RewriteCond %{REQUEST_URI} ^/index.php$ [OR]
+  #RewriteCond %{REQUEST_URI} ^/(user|admin|themes|core|simplesaml|sites|libraries|profiles|modules|node|sitemaps|lp)(/)?(?(1).*|$) [OR]
+  #RewriteCond %{REQUEST_URI} ^/(views|editor|media_browser|toolbar|filter|contextual|history)/.+ [OR]
+  RewriteCond %{REQUEST_URI} ^/node/feedback(/)?(?(1).*|$) [OR]
+  # RewriteCond %{REQUEST_URI} ^/patient-care/making-an-appointment$ [OR]
+  # RewriteCond %{REQUEST_URI} ^/patient-resources/make-an-appointment$ [OR]
+  RewriteCond %{REQUEST_URI} ^/Pages/bio.aspx.* [OR]
+  # SERVE VIRTUAL CLINIC FROM CMS AND ALL OTHER LOCATIONS FROM STEVIE #
+  RewriteCond %{REQUEST_URI} ^/locations/virtual-clinic$ [OR]
+  #RewriteCond %{REQUEST_URI} ^/locations/.* [OR]
+  # Start Primary Care #
+  RewriteCond %{REQUEST_URI} ^/services/primary-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/urgent-care(/)?(?(1).*|$) [OR]
+  # End Primary Care #
+  # Start Center for Women and Children
+  RewriteCond %{REQUEST_URI} ^/services/center-for-women-and-children(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/gynecology(/)?(?(1)(providers|locations|health-library|videos|approaches-to-gynecologic-care|family-planning|gynecologic-cancer|pediatric|urogynecology|robotic-assisted-gynecology|medical-discovery|grand-rounds-videos)|$)?(?(2).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/obstetrics(/)?(?(1)(providers|locations|childbirth-classes|pain-management|midwifery|birth-options|labor-and-delivery|approaches-to-obstetrics|you-and-your-baby|team|specialized-obstetrics|prenatal-diagnosis|lactation-services|grand-rounds-videos|nicu|diabetes-and-pregnancy|hypertension|teen-pregnancy|prematurity-prevention)|$)?(?(2).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/reproductive(/)?(?(1)(providers|locations|services|team|health-library|referrals|single-women-lgbt|medical-discovery)|$)?(?(2).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/cancer-care/cancer-types/breast$ [OR]
+  RewriteCond %{REQUEST_URI} ^/services/trauma-emergency-services/pediatric-trauma-and-burn-services$ [OR]
+  # End Center for Women and Children #
+  # Start Neurosciences #
+  RewriteCond %{REQUEST_URI} ^/services/neurosciences(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/neurological-surgery(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/neurology(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/epilepsy(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/multiple-sclerosis(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/stroke(/)?(?(1).*|$) [OR]
+  # End Neurosciences #
+  # Start Vascular #
+  RewriteCond %{REQUEST_URI} ^/services/vascular(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/vascular-care(/)?(?(1).*|$) [OR]
+  # End Vascular #
+  # Start Urology and Men's Health #
+  RewriteCond %{REQUEST_URI} ^/services/mens-health(/)?(?(1).*|$) [OR]
+  # End Urology and Men's Health #
+  # Start Mental Health Care #
+  RewriteCond %{REQUEST_URI} ^/services/mental-health-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/psychiatry(/)?(?(1).*|$) [OR]
+  # End Mental Health Care #
+  # Start Digestive Health Care #
+  RewriteCond %{REQUEST_URI} ^/services/transplant/liver-transplant(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/digestive-health(/)?(?(1).*|$) [OR]
+  # End Digestive Health Care #
+  # Start Dermatology Care #
+  RewriteCond %{REQUEST_URI} ^/services/dermatology(/)?(?(1).*|$) [OR]
+  # End Dermatology Care #
+  # Start ENT #
+  RewriteCond %{REQUEST_URI} ^/services/ear-nose-and-throat-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/otolaryngology(/)?(?(1).*|$) [OR]
+  # End ENT #
+  # Start Heart Institute #
+  RewriteCond %{REQUEST_URI} ^/services/heart-institute(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/cardiology(/)?(?(1)(providers|locations|referrals-general-cardiology|cardiac-surgery|cardiac-electrophysiology|atrial-fibrillation|interventional-cardiology|adult-congenital-heart-disease|advanced-heart-failure|structural-heart-services|hypertrophic-cardiomyopathy|prosthetic-heart-valve-consult-clinic|cardio-oncology|cardiovascular-genetics)|$)?(?(2).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/transplant/heart-transplant(/)?(?(1).*|$) [OR]
+  # End Heart Institute #
+  # Start Eye Institute #
+  RewriteCond %{REQUEST_URI} ^/services/eye-institute(/)?(?(1).*|$) [OR]
+  # End Eye Institue #
+  # Start Palliative Care #
+  RewriteCond %{REQUEST_URI} ^/services/palliative(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/palliative-care(/)?(?(1).*|$) [OR]
+  # End Palliative Care #
+  # Start Sleep Medicine #
+  RewriteCond %{REQUEST_URI} ^/services/sleep-institute(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/sleep-medicine(/)?(?(1).*|$) [OR]
+  # End Sleep Medicine #
+  # Start Allergy & Immunology #
+  RewriteCond %{REQUEST_URI} ^/services/allergy-immunology(/)?(?(1).*|$) [OR]
+  # End Allergy & Immunology #
+  # Start Laboratory Medicine & Pathology #
+  RewriteCond %{REQUEST_URI} ^/services/laboratory-medicine-and-pathology(/)?(?(1).*|$) [OR]
+  # End Laboratory Medicine & Pathology #
+  # Start Liver Transplant #
+  RewriteCond %{REQUEST_URI} ^/services/liver(/)?(?(1)(providers|locations|transplant|transplant/living-donor-liver-transplant|infectious-disease|patients-families|referrals|transplant-team)|$)?(?(2).*|$) [OR]
+  # End Liver Transplant #
+  # Start Rehab Medicine #
+  RewriteCond %{REQUEST_URI} ^/services/rehabilitation-medicine(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/rehabilitation(/)?(?(1)(providers|locations|neuro-rehab-program|health-library|videos|inpatient|faq|patient-education)|$)?(?(2).*|$) [OR]
+  # End Rehab Medicine #
+  # Start Radiology #
+  RewriteCond %{REQUEST_URI} ^/services/radiology(/)?(?(1)(providers|locations|videos|appointments|radiation-safety)|$)?(?(2).*|$) [OR]
+  # End Radiolology #
+  # Start Cancer Care #
+  RewriteCond %{REQUEST_URI} ^/services/cancer-care(/)?(?(1).*|$) [OR]
+  # End Cancer Care #
+  # Start Orthopedic Surgery and Sports Medicine #
+  RewriteCond %{REQUEST_URI} ^/services/ssoh(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/orthopedic-surgery-sports-medicine(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/foot-ankle(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/hip-knee(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/hes(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/sports-medicine(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/sports-medicine-orthopedic-surgery [OR]
+  # End Orthopedic Surgery and Sports Medicine #
+  # Start Spine Care #
+  RewriteCond %{REQUEST_URI} ^/services/spine(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/spine-care(/)?(?(1).*|$) [OR]
+  # End Spine Care #
+  # Start Burn Care #
+  RewriteCond %{REQUEST_URI} ^/services/burn-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/harborview/services/burn-center(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/trauma-emergency-services/getting-to-harborview(/)?(?(1).*|$) [OR]
+  # End Burn Care #
+  # Start Trauma Care #
+  RewriteCond %{REQUEST_URI} ^/services/trauma-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^harborview/services/trauma(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^harborview/services/trauma-center(/)?(?(1).*|$) [OR]
+  # End Trauma Care #
+  # Start Lung Care #
+  RewriteCond %{REQUEST_URI} ^/services/lung-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/lung(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/transplant/lung-transplant(/)?(?(1).*|$) [OR]
+  # End Lung Care #
+  # Start Reconstructive & Plastic Surgery #
+  RewriteCond %{REQUEST_URI} ^/services/reconstructive-surgery(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/surgery/reconstructive-and-plastic-surgery(/)?(?(1).*|$) [OR]
+  # End Reconstructive & Plastic Surgery #
+  # Start Surgery #
+  RewriteCond %{REQUEST_URI} ^/services/surgery(/)?(?(1).*|$) [OR]
+  # End Surgery #
+  # Start Rheumatology #
+  RewriteCond %{REQUEST_URI} ^/services/rheumatology(/)?(?(1).*|$) [OR]
+  # End Rheumatology #
+  # Start Medical Genetics #
+  RewriteCond %{REQUEST_URI} ^/services/medical-genetics(/)?(?(1).*|$) [OR]
+  # End Medical Genetics #
+  # Start Hematology #
+  RewriteCond %{REQUEST_URI} ^/services/hematology(/)?(?(1).*|$) [OR]
+  # End Hematology #
+  # Start Infectious Diseases #
+  RewriteCond %{REQUEST_URI} ^/services/infectious-diseases(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/patient-care/ebola(/)?(?(1).*|$) [OR]
+  # End Infectious Diseases #
+  # Start Anesthesiology & Pain Medicine #
+  RewriteCond %{REQUEST_URI} ^/services/pain(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/anesthesiology-and-pain-medicine(/)?(?(1).*|$) [OR]
+  # End Anesthesiology & Pain Medicine #
+  # Start Diabetes and Endocrinology #
+  RewriteCond %{REQUEST_URI} ^/services/diabetes-endocrinology(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/diabetes(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/pituitary(/)?(?(1).*|$) [OR]
+  # End Diabetes and Endocrinology #
+  # Start Kidney Care #
+  RewriteCond %{REQUEST_URI} ^/services/kidney-care(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/transplant/kidney-transplant(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/kidney(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/kidney/kidney-genetics(/)?(?(1).*|$) [OR]
+  # End Kidney Care #
+  # Start Transplant #
+  RewriteCond %{REQUEST_URI} ^/services/transplant(/)?(?(1).*|$) [OR]
+  # End Transplant #
+  # Start Entites #
+  # End Entities #
+  # Start Emergency Medicine #
+  RewriteCond %{REQUEST_URI} ^/services/trauma-emergency-services(/)?(?(1).*|$) [OR]
+  RewriteCond %{REQUEST_URI} ^/services/emergency-medicine(/)?(?(1).*|$) [OR]
+  # End Emergency Medicine #
+  # Start Virtual Clinic #
+  RewriteCond %{REQUEST_URI} ^/services/virtual-clinic$
+  # End Virtual Clinic #
+
+  RewriteRule ^(.*)$ - [CO=uwm-proxied-page-visit:true:%{HTTP_HOST}:0:/,C]
+  RewriteRule ^(.*)$ https://cms.uwmedicine.org%{REQUEST_URI} [L,P]
+
+
+
+  ### SERVE EVERYTHING ELSE FROM stevie.cms.uwmedicine.org ###
+  ### This can be removed when cms.uwmedicine.org is decomissioned ###
+  RewriteCond %{HTTP_HOST} ^www.uwmedicine.org
+  RewriteRule ^(.*)$ https://stevie.cms.uwmedicine.org%{REQUEST_URI} [L,P]
+
+</IfModule>
+##
+#
+#
+# END UWMedicine rules
+#
+#
+##
+
 #
 # Apache/PHP/Drupal settings:
 #
@@ -34,7 +671,7 @@ AddEncoding gzip svgz
   php_value mbstring.http_input             pass
   php_value mbstring.http_output            pass
   php_flag mbstring.encoding_translation    off
-  # PHP 5.6 has deprecated $HTTP_RAW_POST_DATA and produces warnings if this is
+  # PHP 5.6 has deprecated$HTTP_RAW_POST_DATA and produces warnings if this is
   # not set.
   php_value always_populate_raw_post_data   -1
 </IfModule>
@@ -150,6 +787,10 @@ AddEncoding gzip svgz
   # Copy and adapt this rule to directly execute PHP files in contributed or
   # custom modules or to run another PHP application in the same directory.
   RewriteCond %{REQUEST_URI} !/core/modules/statistics/statistics.php$
+
+  # UWM-CMS: Allow access to simplesaml paths
+  RewriteCond %{REQUEST_URI} !^/simplesaml
+
   # Deny access to any other PHP files that do not match the rules above.
   # Specifically, disallow autoload.php from being served directly.
   RewriteRule "^(.+/.*|autoload)\.php($|/)" - [F]
@@ -160,12 +801,12 @@ AddEncoding gzip svgz
     # Serve gzip compressed CSS files if they exist and the client accepts gzip.
     RewriteCond %{HTTP:Accept-encoding} gzip
     RewriteCond %{REQUEST_FILENAME}\.gz -s
-    RewriteRule ^(.*)\.css $1\.css\.gz [QSA]
+    RewriteRule ^(.*)\.css$1\.css\.gz [QSA]

     # Serve gzip compressed JS files if they exist and the client accepts gzip.
     RewriteCond %{HTTP:Accept-encoding} gzip
     RewriteCond %{REQUEST_FILENAME}\.gz -s
-    RewriteRule ^(.*)\.js $1\.js\.gz [QSA]
+    RewriteRule ^(.*)\.js$1\.js\.gz [QSA]

     # Serve correct content types, and prevent mod_deflate double gzip.
     RewriteRule \.css\.gz$ - [T=text/css,E=no-gzip:1]
@@ -177,6 +818,17 @@ AddEncoding gzip svgz
       # Force proxies to cache gzipped & non-gzipped css/js files separately.
       Header append Vary Accept-Encoding
     </FilesMatch>
+    #
+    #
+    #
+    # UWM Image caching:
+    <FilesMatch "\.(jpg|jpeg|png|gif|swf)$">
+        # Set images to expire in one year
+        Header set Cache-Control "max-age=31536000, public"
+    </FilesMatch>
+    #
+    #
+    #
   </IfModule>
 </IfModule>

@@ -186,4 +838,5 @@ AddEncoding gzip svgz
   Header always set X-Content-Type-Options nosniff
   # Disable Proxy header, since it's an attack vector.
   RequestHeader unset Proxy
-</IfModule>
\ No newline at end of file
+</IfModule>
+
